Update schema, remove SystemTextJson package, and release 1.0.0-alpha04 (#36)
googleapis/google-cloudevents-dotnet: README.md

[CloudEventConverters](https://github.com/googleapis/google-cloudevents-dotnet/blob/master/src/Google.Events/CloudEventConverters.cs)
class that uses those attributes.

The `Google.Events.Protobuf` contains the Google event data types,
using Protocol Buffers as the serialization framework.

The design allows for other serialization frameworks (e.g.
System.Text.Json or Json.NET) to be supported in the future, each in
a separate package. Frameworks such as the [.NET Functions
Framework](https://github.com/GoogleCloudPlatform/functions-framework-dotnet)

> can depend only on `Google.Events`, but application code can depend
> on whichever package fits in best with their other dependencies. (If
> your application already depends on System.Text.Json, you may want
> to avoid a dependency on Google.Protobuf, or vice versa.)

Reviewer: I would change the tense of this part:
```suggestion
can depend only on `Google.Events`, but application code could depend
on whichever package fits in best with their other dependencies. (If
your application already depended on System.Text.Json, you could be able
to avoid a dependency on Google.Protobuf, or vice versa.)
```

Author: Hmm... that feels a little more awkward to me. Will merge as-is, and we can tweak later. (We may well need to rewrite this entirely of course...)
